services/checks/checks_test.go

```diff
@@ -57,20 +58,22 @@ func TestDownloadChecks(t *testing.T) {
 
 	sqlDB := testdb.Open(t, models.SkipFixtures, nil)
 	db := reform.NewDB(sqlDB, postgresql.Dialect, nil)
+	portalClient, err := platform.NewClient(db)
+	require.NoError(t, err)
+	portalClient.SetAddress(devPortalAddress)
+	portalClient.SetPublicKeys([]string{devPortalPublicKey})
```

Comment:
feels like these two parameters could be part of the constructor.

Reply:
The idea behind that is these methods needed only for tests.

Reply:
BTW refactored